src/test/groovy/com/jvm_bloggers/core/data_fetching/blogs/json_data/JsonBlogsDataFileValidationSpec.groovy

```diff
@@ -73,5 +65,37 @@ class JsonBlogsDataFileValidationSpec extends Specification {
         assert duplicateIds.isEmpty() : "Duplicate bookmarkable IDs found in json files: $duplicateIds"
     }
 
-}
+    @Unroll
+    def "should check that twitter user tag starts with @"() {
+        expect:
+        tag.startsWith("@")
 
+        where:
+        tag << allTwitterUsersTags()
+    }
+
+
+    @Unroll
+    def "should check that twitter user tag is less than 16 characters"() {
```

Comment:
If you use @Unroll, you should also use #parameterName in test method name so we will have more granular names there.